zerver/openapi/zulip.yaml

```diff
@@ -3867,6 +3867,13 @@ paths:
                 allOf:
                   - $ref: "#/components/schemas/BadEventQueueIdError"
                   - description: |
+                      #### BAD_EVENT_QUEUE_ID errors
```

Comment:
Can you confirm the heading works properly?  I'm worried about leading whitespace confusing markdown.

Reply:
I had checked it earlier, and it was indeed working properly.